tools/wopiopen.py

```diff
@@ -11,21 +11,24 @@
 # usage function
 def usage(exitcode):
   '''Prints usage'''
-  print('Usage : ' + sys.argv[0] + ' [-h|--help] <filename> <uid> <gid>')
+  print('Usage : ' + sys.argv[0] + ' [-h|--help] [-m|--mode 1|2|3] <filename> <uid> <gid>')
```

Comment:
mode == viewmode?

Reply:
Yes, but v == verbose. Anyway this `wopiopen` CLI is a kind of internal command for us to quickly test, and soon it is to be replaced by (your) Reva open CLI.

Reply:
As for the other comment, I think we should be consistent even for the internal namings and loggins.

Reply:
OK, I realize that in this command there's no (any more) references to `verbose` (which I typically use for verbose output/logging), so let's use `viewmode` and drop verbose then.